src/app/ngrx/reducers/index.ts

```diff
@@ -6,8 +6,8 @@ import { EpisodeReducer } from './episode.reducer';
 import { PodcastMetricsReducer, PodcastMetricsModel } from './podcast-metrics.reducer';
 import { EpisodeMetricsReducer, EpisodeMetricsModel } from './episode-metrics.reducer';
 import { AccountState, getAccountEntity, getAccountError } from './account.reducer';
-import { PodcastState, getPodcastEntities, getPodcastError } from './podcast.reducer';
-import { EpisodeState, getEpisodeEntities } from './episode.reducer';
+import { PodcastState, getPodcastEntities, getPodcastsLoaded, getPodcastsLoading, getPodcastsError } from './podcast.reducer';
+import { EpisodeModel, EpisodeState, getEpisodeEntities, getEpisodesLoaded, getEpisodesLoading, getEpisodesError } from './episode.reducer';
 import { CustomRouterReducer } from './router.reducer';
 
```

Comment:
selectors should have some kind of logical separation and grouping and not all just live in this index file, maybe when we can get more refactoring done, but right now this index file isn't holding anything up

Reply:
Yeah, agreed.  This file is getting a bit out of control ... but not worth tackling amidst all this other stuff.